Training/PythonWebServer/webserver.py

```diff
@@ -2,16 +2,34 @@
 
 from http.server import SimpleHTTPRequestHandler, HTTPServer
 import sys
+import psutil
+from datetime import datetime
+import pytz
+import json
 
 class SoarWebServer(SimpleHTTPRequestHandler):
 
     def do_POST(self):
-        self.wfile.write("Send your POST response here!")
+        #Send your POST response here!
 
+        time = datetime.now(tz=pytz.timezone('America/Edmonton')).strftime('%H:%M:%S');
+        cpu_count = str(psutil.cpu_count()).ljust(2);
+        cpu_percent = str(psutil.cpu_percent(interval=1)).ljust(5);
+       	virtual_memory_total= str(round(psutil.virtual_memory().total/(2**30), 2)).ljust(5);
+        virtual_memory_used = str(round(psutil.virtual_memory().used/(2**30), 2)).ljust(4);
+
+        pythonData = {  
+	        "Time": time,
+	        "Total CPUs": cpu_count,
+	        "CPU Usage": cpu_percent,
+	        "Total RAM": virtual_memory_total,
+	        "Used RAM": virtual_memory_used
```

Comment:
These attributes are better named without spaces, something more like a variable like `usedRam` so that javascript can access it like a object.

`data.usedRam`